### PR TITLE
Support other language than english

### PR DESCRIPTION
**Pull request by frimtec**
Would like to use pysma with another language than the hardcoded "en-US".

### CODE REVIEW

**Comment by frimtec** [comment]
Fixed all code layout issues.

**Comment by frimtec** [comment]
Fixed all code layout issues in new test.

**Review by rklomp — CHANGES_REQUESTED** [review]
Thanks for the pull request! This looks like a great addition.
See my comment on the test.



Ps. Run 1 pre-commit install1 to set up the git hook scripts.

File: tests/test_init.py [hunk 1]
```diff
@@ -455,3 +455,16 @@ async def test_post_json(self):
...
                 headers={"content-type": "application/json"},
             )
+
+    @patch("pysma._LOGGER.warning")
+    async def test_unsupported_lang(self, mock_warn, mock_aioresponse):  # noqa: F811
+        """Test fallback lang in case requested lang is not available."""
+        mock_aioresponse.get(
+            f"{self.base_url}/data/l10n/de-CH.json",
+            body="",
+        )
```
[the remark this answers is not in the document]
Comment (frimtec) [reply tests/test_init.py:465]: Thanks @rklomp for your seugesstion. Makes totally sence, will change that ASAP.

**Comment by frimtec** [comment]
Dear @rklomp, would it be possible to make a new release of pysma with the merged change?
My idea is to integrate the new language feature into home assistant.
Many thanks in advance
frimtec

**Comment by rklomp** [comment]
Done! Bumped to 0.7.4

**Pull request merged**